pom.data.xml

```diff
@@ -548,6 +552,7 @@
   <modules>
     <module>./sdk/batch/microsoft-azure-batch</module>
     <module>./sdk/applicationinsights/microsoft-azure-applicationinsights-query</module>
+    <module>./sdk/cognitiveservices/ms-azure-cs-inkrecognizer/pom.xml</module>
```

Comment:
The module name is inconsistent with the rest. All other modules start with `microsoft-`. This starts with `ms-`.

Reply:
For some things like cogservice we abbreviated them because of long file path issues. So I'd suggest sticking with the "ms-" for this one and be consistent with the rest of cogservices directory.

Reply:
I don't think length of filepath is an issue because the applicationinsights module above this line has 67 chars and inkrecognizer with `microsoft` is only 56 chars. Also, the longest module name in cogservices is `custom-vision-prediction` which is 66 chars (1 less than applicationinsights).

`microsoft-azure-applicationinsights-query`
`microsoft-azure-cs-inkrecognizer`
`microsoft-azure-cs-customvision-prediction`

Reply:
We abbreviated both ms and cs in all those directory names. We can certainly decide to alter that but it shouldn't be part of this PR. For this PR it should remain consistent with the other cogservices libraries directories.